common/chain.go

```diff
@@ -114,10 +114,6 @@ func IsHeaderSupportedEvmChain(chainID int64) bool {
 		chainID == 56 // bsc mainnet
 }
 
-func (chain Chain) IsKlaytnChain() bool {
```

Comment:
We won't consider Klaytn anymore?

Reply:
Yeah, it isn't used right now. We can add it later on if needed.